src/test/java/api/support/matchers/ScheduledNoticeMatchers.java

```diff
@@ -0,0 +1,66 @@
+package api.support.matchers;
+
+import static api.support.matchers.TextDateTimeMatcher.isEquivalentTo;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.withJsonPath;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.is;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.policy.Period;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeMatchers {
```

Comment:
I like this pattern, thank you for introducing this and the reusable JSON path matcher